pkg/cmd/root.go

```diff
@@ -225,3 +226,15 @@ func init() {
 		}
 	}
 }
+
+func addV2BillingStubs(rootCmd *cobra.Command) {
+	cmd, _, err := rootCmd.Find([]string{"billing"})
+	if err != nil {
+		// silently fail
+		return
+	}
+	rBillingMeterEventSessionCmd := resource.NewResourceCmd(cmd, "meter_event_session")
+	rBillingMeterEventStreamCmd := resource.NewResourceCmd(cmd, "meter_event_stream")
+	resource.NewUnsupportedV2BillingOperationCmd(rBillingMeterEventSessionCmd.Cmd, "create", "/v2/billing/meter_event_session")
+	resource.NewUnsupportedV2BillingOperationCmd(rBillingMeterEventStreamCmd.Cmd, "create", "/v2/billing/meter_event_stream")
```

Comment:
I know these are temporary but would be nice to have a very short test for them such as https://github.com/stripe/stripe-cli/blob/8d33a698892e945948c3f82084c3e5d51c3c7f26/pkg/cmd/root_test.go#L52

Reply:
yeah agreed, added a small test just to confirm the overrides are present